### PR TITLE
feature: menuitem

### PR DESCRIPTION
**Pull request by alimoncul**

### CODE REVIEW

File: ui/menuitem/menuitem.android.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  private _ios: Partial<{ style: Style }> = {
+    style: Style.DEFAULT
+  };
+  private _title: string;
+  private _onSelected: () => void;
+  constructor(params?: Partial<MenuItem>) {
+    super();
+    const self = this;
+    const EventFunctions = {
+      [MenuItemEvents.Selected]: function () {
```
Comment (cenkce) [inline ui/menuitem/menuitem.android.ts:29]: `EventFunctions ` kaldiralim

File: ui/menuitem/menuitem.android.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        self._onSelected?.();
+      }
+    };
+    eventCallbacksAssign(this, EventFunctions);
```
Comment (cenkce) [inline ui/menuitem/menuitem.android.ts:33]: `eventCallbacksAssign ` kaldiralim

File: ui/menuitem/menuitem.ios.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  private _ios: Partial<{ style: Style }> = {
+    style: Style.DEFAULT
+  };
+  private _title: string = '';
+  private _onSelected: () => void;
+  constructor(params?: Partial<MenuItem>) {
+    super();
+    const self = this;
+
+    const EventFunctions = {
```
Comment (cenkce) [inline ui/menuitem/menuitem.ios.ts:24]: `EventFunctions ` kaldiralim

File: ui/menuitem/menuitem.ios.ts [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      [MenuItemEvents.Selected]: function () {
+        self.onSelected?.();
+      }
+    };
+
+    eventCallbacksAssign(this, EventFunctions);
```
Comment (cenkce) [inline ui/menuitem/menuitem.ios.ts:30]: `eventCallbacksAssign ` kaldiralim

**Comment by alimoncul** [comment]
@cenkce updated

**Pull request merged**